site/en/tutorials/load_data/images.ipynb

```diff
@@ -993,7 +994,7 @@
         "id": "vrbjEvaC5XmU"
       },
       "source": [
-        "So build a model wrapped around MobileNet, and use `tf.keras.layers.GlobalAveragePooling2D` to average over those space dimensions, before the output `tf.keras.layers.Dense` layer:"
+        "Because of this output shape, build a model wrapped around MobileNet, using `tf.keras.layers.GlobalAveragePooling2D` to average over the space dimensions before the output `tf.keras.layers.Dense` layer:"
```

Comment:
```suggestion
        "Because of this output shape, build a model wrapped around MobileNet using `tf.keras.layers.GlobalAveragePooling2D` to average over the space dimensions before the output `tf.keras.layers.Dense` layer:"
```